test/test_remap.py

```diff
@@ -142,11 +147,74 @@ def test_overwrite(self):
 
         # Perform remapping
         remap_to_edge_centers = source_grid['v1'].remap.nearest_neighbor(destination_grid=destination_dataset.uxgrid,
-                                                                                  remap_to="nodes")
+                                                                         remap_to="face centers", coord_type='cartesian')
 
         # Assert the remapped data is different from the original data
         assert not np.array_equal(destination_dataset['v1'], remap_to_edge_centers)
 
+    def test_source_data_remap(self):
+        """Test the remapping of all source data positions."""
+
+        # Open source and destination datasets to remap to
+        source_uxds = ux.open_dataset(mpasfile_QU, mpasfile_QU)
+        destination_dataset = ux.open_dataset(gridfile_geoflow, dsfile_v1_geoflow)
+
+        # Remap from `face_centers`
+        face_centers = source_uxds['latCell'].remap.nearest_neighbor(
+            destination_grid=destination_dataset.uxgrid,
```

Comment:
Use `destination_grid` from above change instead of `destination_dataset.uxgrid`.